.github/workflows/e2e-test.yml

```diff
@@ -1,10 +1,10 @@
 name: E2E tests
 
 on:
+  schedule:
+    - cron: '0 */6 * * *'
   pull_request:
     branches:
-      - trunk
-      - develop
       - release/*
```

Comment:
Reading https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags,I see the following:

> For a `pull_request` event, only branches and tags on the base are evaluated.

Based on this line, and the fact that the workflow worked on pull requests to `trunk` and `develop`, I understand that `on.pull_request.<branch>` works based on the branch you want to merge the PR onto.

With this in mind, I'd suggest keeping the `trunk` line to run tests on PRs, which are about to get merged into `trunk`. This includes release PRs, as during deployment, they get merged to `trunk`.

I'm thinking whether we should keep `release/*` here, as generally PRs tend to be merged into `develop`, or `trunk` during patch releases. `release/*` branches are just temporary branches, used to verify the contents of a release shortly before merging them to `trunk`, and don't live long enough for PRs to be opened on top of them.